processor/transformprocessor/internal/logs/functions_test.go

```diff
@@ -10,11 +10,11 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/contexts/ottllog"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/transformprocessor/internal/common"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/ottlfuncs"
 )
 
 func Test_LogFunctions(t *testing.T) {
-	expected := common.Functions[ottllog.TransformContext]()
+	expected := ottlfuncs.StandardFuncs[ottllog.TransformContext]()
 	actual := LogFunctions()
 	require.Equal(t, len(expected), len(actual))
```

Comment:
Could we add a test for the StandardConverters as well?

Reply:
`StandardFuncs` contains all the editors and convertors in OTTL.

Reply:
Converters have also been included in their own function because `internal/filter/filterottl` requires only converters to do its condition work.